Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -310,17 +368,48 @@ void ABenchmarkGymGameModeBase::TickUXMetricCheck(float DeltaSeconds)
 		Constants->UXMetricDelay.IsReady())
 	{
 		bHasUxFailed = true;
-		NFR_LOG(LogBenchmarkGymGameModeBase, Error, TEXT("UX metric has failed. RTT: %.8f, UpdateDelta: %.8f, ActivePlayers: %d"), AveragedClientRTTSeconds, AveragedClientUpdateTimeDeltaSeconds, ActivePlayers);
+		NFR_LOG(LogBenchmarkGymGameModeBase, Error, TEXT("NFR scenario failed: UX metric check. RTT: %.8f, UpdateDelta: %.8f, ActivePlayers: %d"), AveragedClientRTTSeconds, AveragedClientUpdateTimeDeltaSeconds, ActivePlayers);
 	}
 
-	PrintUXMetricTimer -= DeltaSeconds;
-	if (PrintUXMetricTimer < 0.0f)
+	if (PrintMetricsTimer.ShouldPrint())
 	{
-		PrintUXMetricTimer = 10.0f;
 		NFR_LOG(LogBenchmarkGymGameModeBase, Log, TEXT("UX metric values. RTT: %.8f(%d), UpdateDelta: %.8f(%d), ActivePlayers: %d"), AveragedClientRTTSeconds, ValidRTTCount, AveragedClientUpdateTimeDeltaSeconds, ValidUpdateTimeDeltaCount, ActivePlayers);
 	}
 }
 
+void ABenchmarkGymGameModeBase::TickActorCountCheck(float DeltaSeconds)
+{
+	if (!HasAuthority())
+	{
+		return;
+	}
+
+	const UNFRConstants* Constants = UNFRConstants::Get(GetWorld());
+	check(Constants);
+
+	// This test respects the initial delay timer in both native and GDK
+	if (!bHasActorCountFailed &&
+		Constants->ActorCheckDelay.IsReady())
```

Comment:
nit: Newline required here?

Reply:
No I just like it is it separates the conditions to make it readable. Style thing I guess.